alternator/executor.cc

```diff
@@ -1812,10 +1812,18 @@ future<executor::request_return_type> executor::put_item(client_state& client_st
     auto op = make_shared<put_item_operation>(_proxy, std::move(request));
     tracing::add_table_name(trace_state, op->schema()->ks_name(), op->schema()->cf_name());
     const bool needs_read_before_write = op->needs_read_before_write();
+
+    if (!co_await client_state.check_has_permission(auth::command_desc(
+            auth::permission::MODIFY,
+            auth::make_data_resource(op->schema()->ks_name(), op->schema()->cf_name())))) {
+        co_return api_error::access_denied(format(
+            "MODIFY permissions denied on {}.{} by RBAC", op->schema()->ks_name(), op->schema()->cf_name()));
```

Comment:
The user name is missing in the message. Is this on purpose? It could be useful for troubleshooting.